packages/typescript-config-node/node20.json

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 20",
+  "_version": "20.1.0",
+
+  "compilerOptions": {
+    "lib": ["es2023"],
+    "module": "node16",
```

```suggestion
    "module": "NodeNext",
```

For enhanced ESM support